### PR TITLE
Create Connor15518.json

### PR DESCRIPTION
**Pull request by connor15518**

### CODE REVIEW

**Comment by jmedian90** [comment]
Good job! I used yours to check mine hope you don't mind.

**Comment by bbogusla** [comment]
:tada:

**Comment by githubteacher** [comment]
@connor15518 we will need to fix a few things in your pull request before you can merge! Please follow the instructions below for how to format your file. (the `---` at the beginning and end of the file are important and you should not add an `_` at the beginning of each line of the file):

screen-shot-2016-10-11-at-10 01 58-am

When you are all done, @ mention me and I will approve the changes!

File: _pins/Connor15518.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: _Connor15518
```
Comment (githubteacher) [inline _pins/Connor15518.json]: @connor15518 Please add `---` as the first line and remove the `_` at the beginning of your username.

File: _pins/Connor15518.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+githubHandle: _Connor15518 
+latitude: _38.906066
```
Comment (githubteacher) [inline _pins/Connor15518.json]: Please remove the `_` at the beginning of this line and the next line.

File: _pins/Connor15518.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+longitude:_-121.062676
```
Comment (githubteacher) [inline _pins/Connor15518.json]: Please add a new line at the end containing the `---`.

**Pull request merged**